doc/source/cluster/metrics.md

```diff
@@ -67,7 +67,17 @@ ray_dashboard_api_requests_count_requests_total
 
 You can then see the number of requests to the Ray Dashboard API over time.
 
-To stop Prometheus, run `kill <PID>` where `<PID>` is the PID of the Prometheus process that was printed out when you ran the command. To find the PID, you can also run `ps aux | grep prometheus`.
+To stop Prometheus, run the following commands:
+
+```sh
+# case 1: Ray > 2.40
```

Comment:
Just curious, how do we know which Ray version the change will be added?

Reply:
Currently, the CI team is preparing the release candidates for Ray 2.40, so I guess it will be included in Ray 2.41.